atlp-rwanda/atlp-pulse-fn
# Email templates

# PR Description

This PR updates the email templates.

# Description of tasks that were expected to be completed

- [x] Update the invitation email template to match the required.
- [x] Make sure all the link and redirections uses the correct urls.

# How has this been tested?

1. Login as the admin and navigate to the trainees tab.
2. click the invite user button and insert the email of the trainee to be invited.
3. Lastly, check the email you provide on step 2 for the update template.

# Number of Commits

1

# Screenshots (If appropriate)

# Please check this Checklist before you submit your PR:
